scripts/build_sputnikvm.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+set -e
+
+OS='Unknown OS'
+case "$(uname -s)" in
+    Darwin)
+        OS="Mac";;
+
+    Linux)
+        OS="Linux";;
+
+    CYGWIN*|MINGW32*|MSYS*)
+        OS="Windows";;
+
+    *)
+        echo 'Unknown OS'
+        exit;;
+esac
+
+if [ "$OS" == "Windows" ]; then
+    cd %GOPATH%\src\github.com\ethereumproject
+	# Check if git is happening in svm-ffi.
+	if [ -d "%GOPATH%\src\github.com\ethereumproject\sputnikvm-ffi" ]; then
+		cd sputnikvm-ffi
+		if [ -d "%GOPATH%\src\github.com\ethereumproject\sputnikvm-ffi\.git" ]; then
+			remote_name=$(git remote -v | head -1 | awk '{print $1;}')
+			if [ ! "%remote_name%" == "" ]; then
+				echo "Updating SputnikVM FFI from branch [%remote_name%]..."
+				git pull %remote_name% master
+			fi
+		fi
+    	cd c\ffi
+	else
+    	git clone https://github.com/ethereumproject/sputnikvm-ffi.git
+    	cd sputnikvm-ffi\c\ffi
+	fi
+    cargo build --release
+    copy %GOPATH%\src\github.com\ethereumproject\sputnikvm-ffi\c\ffi\target\release\sputnikvm.lib \
```

Comment:
just an idea to consider would be to refactor this to something like

Grab and build all dependencies for installing/building geth with SputnikVM, but don't build or install
```shell
make sputnikvm
```


```shell
make build geth 
make build geth SPUTNIKVM=1 # runs `make sputnikvm` to ensure/update svm dependencies, then builds geth
```

just as a way to maybe keep things DRY/modular (?)

Reply:
I was thinking to make docs and CI builds updates as separate PRs